scarlet/display.py

```diff
@@ -237,45 +240,65 @@ def show_scene(
             mask = None
 
     panel = 0
-    tree = ComponentTree(sources)
-    model = tree.get_model()
-    ax[panel].imshow(img_to_rgb(model, norm=norm, channel_map=channel_map))
-    ax[panel].set_title("Model")
+    if hasattr(sources, "components"):
+        # The list of sources is already a tree, so just use it
+        tree = sources
+    else:
+        tree = ComponentTree(sources)
+    if observation is None:
+        model = tree.get_model()
+        extent = get_extent(tree.bbox)
+    else:
+        model = tree.get_model(frame=observation.frame)
+        extent = get_extent(observation.frame)
+
+    if show_model:
+        ax[panel].imshow(img_to_rgb(model, norm=norm, channel_map=channel_map), extent=extent)
```

Comment:
setting the extent is fine, but we still need to set `origin='lower'` for consistency.

Reply:
I don't think it matters, since I explicitly set `x_lim` and `y_lim` based on `extent`, so it is always consistent.

Reply:
You might be right but then it's a side effect. The `origin` argument is a swith between visualizing a matrix or an image. For reasons God only knows mpl defaults to matrix for a method that is designed to visualize an image.

Reply:
Oh, I see, good catch. In all of my notebooks I always define `origin="lower"` globally when I import matplotlib (notice in the docs, for example https://github.com/pmelchior/scarlet/blob/master/docs/0-quickstart.ipynb?short_path=3c6e710#L27 we do the same). So I'll make this change to make it consistent.